package.json

```diff
@@ -1,34 +1,48 @@
 {
   "private": true,
   "scripts": {
-    "programs:build": "zx ./scripts/program/build.mjs",
-    "programs:test": "zx ./scripts/program/test.mjs",
-    "programs:clean": "zx ./scripts/program/clean.mjs",
-    "programs:format": "zx ./scripts/program/format.mjs",
-    "programs:lint": "zx ./scripts/program/lint.mjs",
-    "solana:check": "zx ./scripts/check-solana-version.mjs",
-    "solana:link": "zx ./scripts/link-solana-version.mjs",
+    "solana:check": "tsx ./scripts/helpers/check-solana-version.mts",
+    "solana:link": "tsx ./scripts/helpers/link-solana-version.mts",
     "generate": "pnpm generate:clients",
-    "generate:clients": "zx ./scripts/generate-clients.mjs",
-    "validator:start": "zx ./scripts/start-validator.mjs",
+    "generate:clients": "tsx ./scripts/helpers/generate-clients.mts",
+    "validator:start": "tsx ./scripts/helpers/start-validator.mts",
     "validator:restart": "pnpm validator:start --restart",
-    "validator:stop": "zx ./scripts/stop-validator.mjs",
-    "clients:js:format": "zx ./scripts/client/format-js.mjs",
-    "clients:js:lint": "zx ./scripts/client/lint-js.mjs",
-    "clients:js:publish": "zx ./scripts/client/publish-js.mjs",
-    "clients:js:test": "zx ./scripts/client/test-js.mjs",
-    "clients:rust:format": "zx ./scripts/client/format-rust.mjs",
-    "clients:rust:lint": "zx ./scripts/client/lint-rust.mjs",
-    "clients:rust:publish": "zx ./scripts/client/publish-rust.mjs",
-    "clients:rust:test": "zx ./scripts/client/test-rust.mjs"
+    "validator:stop": "tsx ./scripts/helpers/stop-validator.mts",
+    "program:build": "tsx ./scripts/rust.mts build-sbf program --features bpf-entrypoint",
+    "program:format": "tsx ./scripts/rust.mts format program",
+    "program:lint": "tsx ./scripts/rust.mts lint program",
+    "program:lint:clippy": "tsx ./scripts/rust.mts lint-clippy program",
+    "program:lint:docs": "tsx ./scripts/rust.mts lint-docs program",
+    "program:lint:features": "tsx ./scripts/rust.mts lint-features program",
+    "program:test": "tsx ./scripts/rust.mts test program",
```

Comment:
Is there a reason this is inconsistent with the other repos which prefix with `programs` instead of `program`? I'm asking because there are monorepo jobs which test downstream repos, and it's expecting to be able to do `programs:build` in all them, for example

Reply:
Same here, I think we renamed to `program` in the last refactoring since we usually have a single program per repository.